keyboards/eve/meteor/keymaps/via/rules.mk

```diff
@@ -0,0 +1,6 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
+
+# build options
+MOUSEKEY_ENABLE = no
+EXTRAKEY_ENABLE = no
```

Comment:
```suggestion
```

Reply:
Unfortunately, this will not compile (as is) for the atmega32a without this change. See previous rules.mk discussion.

Reply:
It will since #9020, and highly likely your repo is stale.

Reply:
#9020 only seems to be in the development branch, unless I am mistaken. Should I be merging to there, or can we make this a followup change after #9020 gets merged back to master?

Reply:
In any case, I synced and updated submodules, but still have the same issue.

Reply:
#10476 wouldnt have been able to be merged if that were the case.